src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java

```diff
@@ -263,6 +263,9 @@ private static void nullSafeAdd(final Deque<Object> toVisit, final Object o) {
                         LOG.error("Security settings prevent Ehcache from accessing the subgraph beneath '{}'" +
                                   " - cache sizes may be underestimated as a result", field, e);
                         continue;
+                    } catch (RuntimeException e) {
+                        LOG.warn("The JVM is preventing Ehcache from accessing the subgraph beneath '{}'" +
+                                " - cache sizes may be underestimated as a result", field, e);
```

Comment:
Missing the `continue`... probably missing from my original example code.

Reply:
fixed